### PR TITLE
New test

### PR DESCRIPTION
**Pull request by onedebos**

### CODE REVIEW

**Comment by onedebos** [comment]
Completing this project alone as my coding partner and I had some issues. Willow has given us permission to work alone.

File: spec/game_spec.rb [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+RSpec.describe(Game) do
+  let(:game) { Game.new('Mike', 'John') }
+
+  describe '#check_for_win' do
+    it 'returns true if player has winning moves' do
+      game.player1.moves = '123'
+      expect(game.check_for_win(game.player1)).to eql(true)
+    end
+    it 'returns true if player has winning moves' do
```
Comment (bellom) [inline spec/game_spec.rb:14]: Leave one empty line around it/specify blocks. This helps to separate the expectations from their conditional logic (contexts for instance).
Checkout this link for more clarification. :point_right: https://github.com/rubocop-hq/rspec-style-guide#empty-lines-around-examples

kindly apply to the rest of the `it` block in your code.

**Review by bellom — CHANGES_REQUESTED** [review]
Hi @onedebos,
Great work using `Git Flow` and awesome job writing the test, specs are failing if i break some code in tested class.
Just one thing to do; kindly attend to my comment.

_Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you_

**Review by maelfosso — APPROVED** [review]
# Good job :+1: 

You can merge now.
Good luck with the next project.

**Pull request merged**